enclave/sgx/attester.c

```diff
@@ -160,6 +146,7 @@ static oe_result_t _get_evidence(
     if (format_type == SGX_FORMAT_TYPE_LOCAL ||
         format_type == SGX_FORMAT_TYPE_REMOTE)
     { // Evidence of these types has its custom claims hashed.
+        oe_report_header_t* header = NULL;
```

Comment:
Let's not change evidence generation in this release. Old verifiers will not be able to verify new format.

Reply:
As described previously, let's keep the current evidence generation and verification: keep the `oe_report_header_t` header in SGX evidence generated by `oe_get_evidence()`.